src/app/challengeDetailsScreen.tsx

```diff
@@ -0,0 +1,61 @@
+import { StyleSheet, Text, View } from "react-native";
+import { useLocalSearchParams } from "expo-router";
+
+import ProgressBar from "@src/components/ui/ProgressBar";
+import Stack from "@src/components/ui/Stack";
+
+const challengeDetails = () => {
+  const params = useLocalSearchParams();
+
+  const name = params.name;
```

Comment:
This page should be receiving the id and making an api call to get the rest of the info.

Reply:
As discussed offline, this will be implemented in the next PR. For now, I added a TODO.